0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -101,13 +103,17 @@ The request must specify either `source_amount` or `destination_amount` but not
   "method": "quote_response",
   "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
   "data": {
-    "source_connector_account": "mark",
```

Comment:
> is "relative ILP address" even a thing?

I would assume so BUT we should be careful about mixing that up with ledger specific account identifiers. I'm not sure which one this is?